.travis.yml

```diff
@@ -1,9 +1,21 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
+  - "0.12"
+env:
+  global:
+    # Please get your own free key if you want to test yourself
+    - BROWSERSTACK_USERNAME: dtktestaccount1
```

Comment:
According to the [Travis docs](http://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings), if you define the variables in the Travis interface, they aren't available to untrusted builds (PRs).